### PR TITLE
Single Product Page

### PR DESCRIPTION
**Pull request by sakshi006**
1. Made single product page
2. Used cart API

### CODE REVIEW

**Review by Asuraking1n — APPROVED** [review]
Code structure looks good,
UI is also good

**Review by anurag-kkumarr — APPROVED** [review]
Code structure is clean
user interface also looks good
looks good to merge.

File: src/Pages/Home/Home.jsx [hunk 1]
```diff
@@ -1,14 +1,15 @@
 import React from "react";
 import { Link } from "react-router-dom";
-
-import { homePageProducts } from "../../Assets/HomePageData/Homepagedata";
 import { homePageGeneral } from "../../Assets/HomePageData/HomePageGeneral";
 
 import "./Home.css";
 import headerImg1 from "../../Assets/head2.jpg";
```
Comment (sudosohan) [inline src/Pages/Home/Home.jsx:6]: Remove this if you don't need

File: src/Pages/SingleProduct/SingleProduct.css [hunk 2]
```diff
@@ -0,0 +1,76 @@
+.single-product-main{
+  display: flex;
```
Comment (sudosohan) [inline src/Pages/SingleProduct/SingleProduct.css:2]: Just a suggestion to use a grid layout to make your layout look better in desktop screens

**Review by sohan9819 — APPROVED** [review]
Everything looks good to me 👍

**Pull request merged**